liquibase-core/src/main/java/liquibase/exception/CommandValidationException.java

```diff
@@ -6,7 +6,11 @@
 public class CommandValidationException extends CommandExecutionException {
 
     public CommandValidationException(String argument, String message) {
-        super("Invalid argument '" + argument + "': " + message);
+        this(argument, message, false);
+    }
+
+    public CommandValidationException(String argument, String message, boolean hasInitProjectParameters) {
```

Comment:
hasInitProjectParameters as an argument here seems out of place here. As a generic extension, it shouldn't be containing init related messaging?

Reply:
Good point. I moved this logic up a layer.